msteams-platform/messaging-extensions/how-to/link-unfurling.md

```diff
@@ -16,7 +16,9 @@ This document guides you on how to add link unfurling to your app manifest using
 > * Currently, link unfurling is not supported on Mobile clients.
 > * The link unfurling result is cached for 30 minutes.
 
-The Azure DevOps messaging extension uses link unfurling to look for URLs pasted into the compose message area pointing to a work item. In the following image, a user has pasted a URL for a work item in Azure DevOps, which the messaging extension has resolved into a card:
+When a URL is pasted in the compose message area, the web service **Unfurls** the URL into a detailed card.
+
+ The following image displays link unfurling feature when a link is pasted in messaging extension:
```

Comment:
Why has this text been updated? The original information is right. Please revert. It doesn't seem to be part of the issue.